packages/twenty-emails/src/emails/clean-inactive-workspaces.email.tsx

```diff
@@ -1,46 +1,62 @@
+import { i18n } from '@lingui/core';
+import { t } from '@lingui/core/macro';
+import { Plural, Trans } from '@lingui/react/macro';
+
 import { BaseEmail } from 'src/components/BaseEmail';
 import { CallToAction } from 'src/components/CallToAction';
 import { HighlightedText } from 'src/components/HighlightedText';
 import { MainText } from 'src/components/MainText';
 import { Title } from 'src/components/Title';
+import { loadAndActivateLocale } from 'src/utils/loadAndActivateLocale';
```

Comment:
On the server side I'm not sure it makes as much sense to do dynamic imports, we should probably just do static imports of all locales in BaseEmail.tsx

Reply:
That way we'd get rid of the async/await which feels a bit strange